lib/statix.ex

```diff
@@ -335,8 +335,9 @@ defmodule Statix do
   @doc false
   def transmit(conn, type, key, val, options)
       when (is_binary(key) or is_list(key)) and is_list(options) do
-    sample_rate = Keyword.get(options, :sample_rate)
+    sample_rate = Keyword.get(options, :sample_rate, Application.get_env(:statix, :sample_rate))
```

Comment:
I think instead of fetching the value from env each call we should read it once and store in the connection struct.

Reply:
Looks reasonable. Let me update PR and see if I'm on right track.